.github/workflows/test_periodic.yml

```diff
@@ -36,14 +36,6 @@ jobs:
         echo "GOPATH=$GOPATH" >> $GITHUB_ENV
         echo "$GOPATH/bin" >> $GITHUB_PATH
 
-    - name: Verify
-      run: |
-        pushd go-controller
```

Comment:
this code comes from the repo, we don' t need to run the verify, it is assumed that if is merged the code is correct